src/app.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable **all** linter check on your code to ensure that your code is quality and robust.

Reply:
It is necessary for me to do that because one of the requirements was that we use classes and the linters check was flagging my code for using classes instead of a stateless function. I also informed the reviewer of the next project about this and he took note of it. I will make sure I do that again for my next submission